src/chapters/control-flow/loops.rst

```diff
@@ -0,0 +1,278 @@
+Loops
+=====
+
+.. index:: ! for loop
+
+``for`` Loop
+------------
+
+In Java we write a definite loop (aka a **for loop**) as:
+
+.. sourcecode:: java
+   :linenos:
+
+   for (int i = 0; i < 10; i++ ) {
+      System.out.println(i);
+   }
+
+.. note:: 
+
+   You may not be familiar with the expression ``i++`` since it is not
+   found in all languages. The ``++`` is an increment operator that has the same
+   effect as ``i += 1``. In this example, since the ``++`` comes after
+   ``i``, we call it a postfix increment operator. There is also a ``--``
+   decrement operator in Java. For more information, see 
+   `Increment and Decrement Operators <http://www.javawithus.com/tutorial/increment-and-decrement-operators>`__.
+
+
+The Java ``for`` loop gives you explicit control over the starting, stopping, and stepping 
+of the loop variable inside the parentheses. You can think of it this way:
+
+.. sourcecode:: java
+   :linenos:
+
+   for (start clause; stop clause; step clause) {
+      statement1
+      statement2
+      ...
+   }
+
+If you want to start at 100, stop at 0 and count backward by 5, the loop is written as:
+
+.. sourcecode:: java
```

Comment:
This one might be longer than we want for including an output block, but adding it would be consistent with our other examples.

**Output**
100
95
90
etc.